diff -urN aa/gail-1.5.6/configure.in gail-1.5.6/configure.in
--- aa/gail-1.5.6/configure.in	2004-02-25 13:41:03.000000000 +0100
+++ gail-1.5.6/configure.in	2004-02-25 16:52:23.810272904 +0100
@@ -90,7 +90,7 @@
 	$GTK_PACKAGES >= $GTK_REQUIRED_VERSION \
 	$LIBGNOMECANVAS_PACKAGES >= $LIBGNOMECANVAS_REQUIRED_VERSION)
 
-ALL_LINGUAS="ar az be ca cs cy de el en_CA es fa fi fr ga hr id it ja ko lt mn ms nl nn no pt sq sr sr@Latn sv uk"
+ALL_LINGUAS="ar az be ca cs cy de el en_CA es fa fi fr ga hr id it ja ko lt mn ms nb nl nn pt sq sr sr@Latn sv uk"
 AM_GLIB_GNU_GETTEXT
 
 AM_GLIB_DEFINE_LOCALEDIR(GAIL_LOCALEDIR)
