--- gail-1.7.0/configure.in.orig	2004-08-20 09:19:18.000000000 +0200
+++ gail-1.7.0/configure.in	2004-08-20 19:19:25.321069392 +0200
@@ -90,7 +90,7 @@
 	$GTK_PACKAGES >= $GTK_REQUIRED_VERSION \
 	$LIBGNOMECANVAS_PACKAGES >= $LIBGNOMECANVAS_REQUIRED_VERSION)
 
-ALL_LINGUAS="ar az be bg bn bs ca cs cy da de el en_CA en_GB es et fa fi fr ga gu hi hr hu id is it ja ko lt mi mn ms nb ne nl nn no pa pl pt pt_BR ro ru sq sr sr@Latn sv ta tk tr uk zh_CN zh_TW"
+ALL_LINGUAS="ar az be bg bn bs ca cs cy da de el en_CA en_GB es et fa fi fr ga gu hi hr hu id is it ja ko lt mi mn ms nb ne nl nn pa pl pt pt_BR ro ru sq sr sr@Latn sv ta tk tr uk zh_CN zh_TW"
 AM_GLIB_GNU_GETTEXT
 
 AM_GLIB_DEFINE_LOCALEDIR(GAIL_LOCALEDIR)
